.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule ".scripts"]
+	path = .scripts
+	url = https://github.com/libero/scripts
```

Comment:
spacing seems a little wide here. Is that intentional?

Reply:
I think `git submodule` creates this with tabs and those should be replaced by 4 spaces